packages/date-time-utilities/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@fluentui/date-time-utilities",
+  "version": "7.0.0",
+  "description": "Date-time utilities",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "test": "just-scripts test",
+    "just": "just-scripts",
+    "code-style": "just-scripts code-style",
+    "clean": "just-scripts clean",
+    "start-test": "just-scripts jest-watch",
+    "lint": "just-scripts lint",
+    "update-api": "just-scripts update-api"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.9.0",
```

Should be like this to match the other packages
```suggestion
    "@types/jest": "~24.9.0",
```